docs/places.mdown

```diff
@@ -0,0 +1,107 @@
+Rust Places
```

Comment:
µnit: everything else uses .md for markdown files, so this should too (even just to make globbing and such easier)